addons/mail/static/src/messaging/component/thread_needaction_preview/thread_needaction_preview.scss

```diff
@@ -0,0 +1,116 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_ThreadNeedactionPreview {
+    @include o-mail-notification-list-item-layout();
+
+    &:hover .o_ThreadNeedactionPreview_markAsRead {
+        opacity: 1;
+    }
+}
+
+.o_ThreadNeedactionPreview_content {
+    @include o-mail-notification-list-item-content-layout();
+}
+
+.o_ThreadNeedactionPreview_core {
+    @include o-mail-notification-list-item-core-layout();
+}
+
+.o_ThreadNeedactionPreview_coreItem {
+    @include o-mail-notification-list-item-core-item-layout();
+}
+
+.o_ThreadNeedactionPreview_counter {
+    margin: map-get($spacers, 0) map-get($spacers, 2);
```

Comment:
Everything should have its own mixin, the only reason why it wasn't the case for some is because they were not used for notification request (which was the "first duplicate").

But now this style is used for both thread previews and probably also for notification group, so it should have mixin too.

I was lazy to do it on the notif PR because it was too big, but now would be appropriate.